sled-agent/src/sim/sled_agent.rs

```diff
@@ -207,7 +174,7 @@ impl SledAgent {
             updates: UpdateManager::new(config.updates.clone()),
             nexus_address,
             nexus_client,
-            disk_id_to_region_ids: Mutex::new(HashMap::new()),
+            simulated_upstairs: simulated_upstairs.clone(),
```

Comment:
Nit - can we drop this `.clone()`?

```suggestion
            simulated_upstairs,
```

Reply:
yep - 79224960f